.github/workflows/e2e.yml

```diff
@@ -5,7 +5,7 @@ on: pull_request
 jobs:
     test:
         name: E2E Tests
-        runs-on: ubuntu-18.04
+        runs-on: ubuntu-latest
```

Comment:
What is the motivation for this? It looks like it could bite us when new ubuntu versions are released.

Reply:
The idea is to catch issues early and have a CI environment that is closer to a production environment. 🙂